simple-phpunit

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env php
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+// Please update when phpunit needs to be reinstalled with fresh deps:
+// Cache-Id-Version: 2017-11-22 09:30 UTC
+
+error_reporting(-1);
+
+$getEnvVar = function ($name, $default = false) {
```

Comment:
this file looks wrong. Install the phpunit-bridge instead of duplicating its code.

Reply:
Temporary code - debugging a weird Travis-only phpunit issue :/